tests/globalhelper/helper.go

```diff
@@ -277,3 +278,24 @@ func AllowAuthenticatedUsersRunPrivilegedContainers() error {
 
 	return nil
 }
+
+// CopyFiles copy file from source to destination.
+func CopyFiles(src string, dst string) error {
+	originalFile, err := os.Open(src)
+	if err != nil {
+		return err
```

Comment:
Please, add more context to this error: `return fmt.Errorf("failed to open file %s: %w", src, err)`

Reply:
thanks